frontend/components/product-list/ProductListDeviceNavigation.tsx

```diff
@@ -0,0 +1,57 @@
+import { Flex } from '@chakra-ui/react';
+import { SecondaryNavbarItem, SecondaryNavbarLink } from '@components/common';
+import { IFIXIT_ORIGIN } from '@config/env';
+import { ProductList } from '@models/product-list';
+import snakeCase from 'lodash/snakeCase';
+import NextLink from 'next/link';
+
+export interface ProductListDeviceNavigationProps {
+   productList: ProductList;
+}
+
+export function ProductListDeviceNavigation({
+   productList,
+}: ProductListDeviceNavigationProps) {
+   const isRootProductList = productList.ancestors.length === 0;
+   let guideUrl: string | undefined;
+   let answersUrl: string | undefined;
+   if (isRootProductList) {
+      guideUrl = `${IFIXIT_ORIGIN}/Guide`;
+      answersUrl = `${IFIXIT_ORIGIN}/Answers`;
+   } else if (productList.deviceTitle && productList.deviceTitle.length > 0) {
+      guideUrl = `${IFIXIT_ORIGIN}/Device/${snakeCase(
+         productList.deviceTitle
+      )}`;
+      answersUrl = `${IFIXIT_ORIGIN}/Answers/Device/${snakeCase(
+         productList.deviceTitle
+      )}`;
```

Comment:
I don't think we need to change the case here, but we will want to replace space with underscore.

Reply:
@sterlinghirsh fixed 👍